tools/ZydisInfo.c

```diff
@@ -777,6 +810,10 @@ static void PrintAVXInfo(const ZydisDecodedInstruction* instruction)
             ZydisRegisterGetString(instruction->avx.mask.reg),
             CVT100_OUT(COLOR_VALUE_LABEL), CVT100_OUT(COLOR_VALUE_B),
             strings_mask_mode[instruction->avx.mask.mode], CVT100_OUT(COLOR_VALUE_LABEL));
+        if (instruction->attributes & ZYDIS_ATTRIB_HAS_SCC)
```

Comment:
We should probably put this in a dedicated APX section together with the "zu" (which is not implemented yet at all).